doc/install-hardware-platforms.md

```diff
@@ -0,0 +1,3 @@
+
+# Installation on different hardware platforms
```

Comment:
Why do we need this file?

Reply:
I don't remember anymore. I'll remove it.